src/user.spec.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-env jest */
+
+import {
+    getUser,  
+    testConnection,
+    xo,
+  } from "./util";
+  
+  describe("user", () => {
+    let userId1, userId2;
+  
+    test("getAll() works", async () => {
+      expect.assertions(1);
+      const users = await xo.call("user.getAll");
+      expect(users).toBeInstanceOf(Array);
+    });
+  
+    describe(".create()", () => {
+      test("create user <email, password>", async () => {
+        expect.assertions(2);
+  
+        userId1 = await xo.call("user.create", {
+          email: "wayne1@vates.fr",
+          password: "batman1",
+        });
+        expect(typeof userId1).toBe("string");
+  
+        const user = await getUser(xo, userId1);
+        expect(user).toEqual(
```

Comment:
Omit the user Id and use `toMatchSnapshot`

Reply:
IMHO, this test should be in the `getAll()` block.

Reply:
So, in the test ` .create() `, we have only the test on the typeof id ?

Reply:
After thought, the code will be more complicated if we isolate the tests.

So, we can let the test of the created user properties and the connection in the `.create()` block, but IMHO, better do it in a separated `test()`  for better visibility.
![image](https://user-images.githubusercontent.com/21563339/49926031-ebe7a880-feba-11e8-8d65-6fb9d32b1155.png)

What do you think?

Reply:
I think it's better :+1: 

But I want to be sure : 
- creation for the test on typeof id
- properties for the test with getAll()
- connection (for the connection).

Reply:
Yes :+1: